pkg/pushsync/pb/pushsync.proto

```diff
@@ -10,3 +10,7 @@ message Delivery {
   bytes Address = 1;
   bytes Data = 2;
 }
+
+message Receipt {
+  bytes Address = 1;
```

Comment:
please rebase, this is already on master, not sure why this is showing up on this PR

Reply:
oh, I was building it for Zahoor and did not notice I commited to my branch :)